Conch/lib/Conch.pm

```diff
@@ -147,7 +147,7 @@ sub startup {
 			my ( $c, $args ) = @_;
 			return unless my $template = $args->{template};
 			if ( $template =~ /exception/ ) {
-				my $exception = $args->{exception};
+				my $exception = $c->stash('exception');
```

Comment:
Is it *always* in the stash? i.e. we don't have to do:

    my $exception = $c->stash('exception') // $args->{exception};

?

Reply:
I'll approve this now so you can merge whenever.

Reply:
In my testing, yes, it's always in the stash. I can add that safety valve if you'd like though